### DISCUSSION TITLE
Why has localization been removed?

### DISCUSSION THREAD

**Discussion by Chunbear** [vladmandic/sdnext#96] [Q&A]
When I tried to install the localization language pack, I found that it didn't work.
So when I looked closer I found that the readme says: [Drops localizations]
I don't understand why this feature should be dropped, many users in other languages are using it.

**Comment by vladmandic** [comment]
there were bugs around it and i don't have any resources to maintain it. simply question of priorities.
if there is an interested maintainer, i'd welcome bringing it back.

**Reply by Chunbear** [reply]
Doesn't the localization feature just require an extension to be installed? Or does it take a lot of time to maintain to implement this feature of localization?

**Reply by vladmandic** [reply]
localization happens in runtime by replacing text fields on the fly in javascript, downloaded packages are just strings
